CRITICAL - 2022-12-10 06:45:13 --> syntax error, unexpected end of file, expecting '('
in APPPATH\Helpers\Global_helper.php on line 205.
 1 SYSTEMPATH\Controller.php(85): helper([...])
 2 APPPATH\Controllers\BaseController.php(46): CodeIgniter\Controller->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 3 SYSTEMPATH\CodeIgniter.php(867): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 4 SYSTEMPATH\CodeIgniter.php(448): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-10 07:23:38 --> mysqli_sql_exception: Table 'pixel.block' doesn't exist in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `Bl...', 0)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Bl...')
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Bl...')
#3 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Bl...', Array, false)
#4 C:\xampp\htdocs\test\da_sein\app\Controllers\Admin\Block.php(76): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Admin\Block->create_action()
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Block))
#7 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-12-10 07:25:32 --> mysqli_sql_exception: Table 'pixel.block' doesn't exist in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `Bl...', 0)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Bl...')
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Bl...')
#3 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Bl...', Array, false)
#4 C:\xampp\htdocs\test\da_sein\app\Controllers\Admin\Block.php(76): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Admin\Block->create_action()
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Block))
#7 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-12-10 07:25:55 --> mysqli_sql_exception: Table 'pixel.block' doesn't exist in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `Bl...', 0)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Bl...')
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Bl...')
#3 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Bl...', Array, false)
#4 C:\xampp\htdocs\test\da_sein\app\Controllers\Admin\Block.php(76): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Admin\Block->create_action()
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Block))
#7 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-12-10 07:26:53 --> mysqli_sql_exception: Table 'pixel.block' doesn't exist in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `Bl...', 0)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Bl...')
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Bl...')
#3 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Bl...', Array, false)
#4 C:\xampp\htdocs\test\da_sein\app\Controllers\Admin\Block.php(76): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Admin\Block->create_action()
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Block))
#7 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-12-10 07:27:03 --> mysqli_sql_exception: Table 'pixel.block' doesn't exist in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `Bl...', 0)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Bl...')
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Bl...')
#3 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Bl...', Array, false)
#4 C:\xampp\htdocs\test\da_sein\app\Controllers\Admin\Block.php(76): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Admin\Block->create_action()
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Block))
#7 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-12-10 07:27:06 --> mysqli_sql_exception: Table 'pixel.block' doesn't exist in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `Bl...', 0)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Bl...')
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Bl...')
#3 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Bl...', Array, false)
#4 C:\xampp\htdocs\test\da_sein\app\Controllers\Admin\Block.php(76): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Admin\Block->create_action()
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Block))
#7 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-12-10 07:27:21 --> mysqli_sql_exception: Table 'pixel.block' doesn't exist in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `Bl...', 0)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Bl...')
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Bl...')
#3 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Bl...', Array, false)
#4 C:\xampp\htdocs\test\da_sein\app\Controllers\Admin\Block.php(76): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Admin\Block->create_action()
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Block))
#7 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-12-10 08:18:53 --> mysqli_sql_exception: Table 'pixel.block' doesn't exist in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `Bl...', 0)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Bl...')
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Bl...')
#3 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Bl...', Array, false)
#4 C:\xampp\htdocs\test\da_sein\app\Controllers\Admin\Block.php(76): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Admin\Block->create_action()
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Block))
#7 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-12-10 08:19:44 --> mysqli_sql_exception: Table 'pixel.block' doesn't exist in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `Bl...', 0)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Bl...')
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Bl...')
#3 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Bl...', Array, false)
#4 C:\xampp\htdocs\test\da_sein\app\Controllers\Admin\Block.php(76): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Admin\Block->create_action()
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Block))
#7 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-12-10 08:19:55 --> mysqli_sql_exception: Table 'pixel.block' doesn't exist in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `Bl...', 0)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Bl...')
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Bl...')
#3 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Bl...', Array, false)
#4 C:\xampp\htdocs\test\da_sein\app\Controllers\Admin\Block.php(76): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Admin\Block->create_action()
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Block))
#7 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-12-10 22:08:16 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'pixel', 3306, '', 0)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\test\da_sein\app\Controllers\Home.php(19): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Home->index()
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\test\da_sein\app\Controllers\Home.php(19): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Home->index()
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-12-10 22:08:16 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `slider`', [], false)
 3 APPPATH\Controllers\Home.php(19): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
